src/components/shared/regionfilter/housetable/index.jsx

```diff
@@ -0,0 +1,155 @@
+      pagination={paginationFactory({
+        custom: true,
+        page,
+        sizePerPage,
+        totalSize,
+      })}
+    >
+      {({ paginationProps, paginationTableProps }) => (
+        <div>
+          <div>
+            <PaginationListStandalone {...paginationProps} />
+          </div>
+          <BootstrapTable
+            remote
+            keyField='hash'
+            data={data}
+            columns={[
+              {
+                dataField: "hash",
+                hidden: true,
+              },
+              {
+                dataField: "street",
+                text: "street",
+                sort: true,
+              },
+              {
+                dataField: "number",
+                text: "number",
+                sort: true,
+              },
+              {
+                dataField: "postcode",
+                text: "postalcode",
+                sort: true,
+              },
+              {
+                dataField: "region",
+                text: "region",
+                sort: true,
+              },
+              {
+                dataField: "city",
+                text: "city",
+                sort: true,
+              },
+              {
+                dataField: "coordinates",
+                text: "coordinates",
+                sort: true,
+              },
+            ]}
+            onTableChange={onTableChange}
+            {...paginationTableProps}
+          />
+        </div>
+      )}
+    </PaginationProvider>
+  </div>
+);
+
+class houseTable extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      streetname: this.props.streetname,
+      items: [],
+      allpoints: [],
+      page: 1,
+      totalSize: 0,
+      currentcity: this.props.currentcity,
+      sizePerPage: 10,
+      updatelist: this.props.updatelist,
+    };
+  }
+  handleTableChange = async (type, { page, sizePerPage }) => {
+    setTimeout(() => {
+      this.fetchData(page, sizePerPage);
+      this.setState(() => ({
+        page,
+        sizePerPage,
+      }));
+    }, 1000);
+  };
+  componentDidUpdate(prevprops) {
+    if (prevprops.streetname !== this.props.streetname) {
+      this.fetchData();
+    }
+  }
+
+  async fetchData(
+    page = this.state.page,
+    sizeperPage = this.state.sizePerPage
+  ) {
+    console.log(this.state.streetname);
+    await Axios.get(ApiActions.StreetInfo, {
+      params: {
+        streetname: this.props.streetname,
+        city: this.props.currentcity,
+        page: page,
+      },
+    })
+      .then((result) => {
+        this.state.updatelist(result.data.houses);
+
+        this.setState({
+          items: result.data.houses,
+          totalsize: result.data.totalStreet,
+        });
+      })
+      .catch((result) => {
+        console.log("error loading results");
+      });
+  }
+
+  componentDidMount() {
+    this.fetchData(this.state.page);
+  }
+  render() {
+    const { t } = this.props;
```

Comment:
```suggestion
```
Volgens sonarcloud wordt deze var nooit gebruikt, klopt dat?

Reply:
@BramHouben Dit is jouw commit, kan je het uitleggen?

Reply:
Ja ik kreeg die translation niet gefixt in deze pagina